src/backend-ghc/Language/Haskell/Tools/BackendGHC/Modules.hs

```diff
@@ -125,7 +125,8 @@ trfLanguagePragma :: Located String -> Trf (Ann AST.UFilePragma (Dom r) RangeSta
 trfLanguagePragma lstr@(L l _) = annLocNoSema (pure l) (AST.ULanguagePragma <$> makeList ", " (pure $ srcSpanStart $ getLoc $ last pragmaElems)
                                                                                               (mapM (trfLocNoSema (pure . AST.ULanguageExtension)) extensions))
   where pragmaElems = splitLocated lstr
-        extensions = filter ((\sp -> srcSpanStart sp /= srcSpanEnd sp) . getLoc)
+        extensions = filter (all isLetter . unLoc)
```

Comment:
This breaks everything. I don't understand what this line of code EXACTLY does, but I guess you wanted to filter empty strings as well. With this change, 1/3 of my tests failed.

Reply:
Now I understand why this line of code was added. You wanted to filter the commas. Still, it isn't working. I will fix this issue in the OrganizeExtensions module. Please fix it here.